config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root to: 'pages#index'
+  get 'mentors/index'
 end
```

Comment:
I don't think this will work... I think it needs to be ```get 'mentors/index' => 'mentors#index'``` ??

Reply:
it does work, but now I'm looking at it and I don't remember why I did or how I knew.... I think I just finished another rails thing and did it the same way.... let me look , but if I find whatever I got it from I will change it.

Reply:
Oh it's the way I generated the controller - it created the route automatically --  It works... but should we change it to what we are used to?

what I did got generate a controller with an ' index route and action (and I think view) generated:
**$ rails generate controller Mentors index**

Reply:
i'm going to change it -- it works for r   url/mentors/index  -- but  I think it should just be url/mentors to get the index